ejb3/src/main/java/org/jboss/as/ejb3/subsystem/ApplicationSecurityDomainDefinition.java

```diff
@@ -96,7 +96,13 @@
             .setRestartAllServices()
             .build();
 
-    private static final AttributeDefinition[] ATTRIBUTES = new AttributeDefinition[] { SECURITY_DOMAIN, ENABLE_JACC };
+    static final SimpleAttributeDefinition LEGACY_COMPLIANT_PRINCIPAL_PROPAGATION = new SimpleAttributeDefinitionBuilder(EJB3SubsystemModel.LEGACY_COMPLIANT_PRINCIPAL_PROPAGATION, ModelType.BOOLEAN, true)
+            .setDefaultValue(ModelNode.TRUE)
+            .setMinSize(1)
```

Comment:
Looks like `setMinSize` could be removed since this attribute is a boolean value.

Reply:
@fjuma fixed